content/v2.0/reference/flux/stdlib/experimental/geo/shapedata.md

```diff
@@ -0,0 +1,123 @@
+---
+title: geo.shapeData() function
+description: >
+  The `geo.shapeData()` function restructures data with existing latitude and longitude
+  fields to meet the requirements of the Geo package by renaming the existing latitude
+  and longitude fields to **lat** and **lon** and adding an **s2_cell_id** tag.
+menu:
+  v2_0_ref:
+    name: geo.shapeData
+    parent: Geo
+weight: 401
+v2.0/tags: [functions, geo]
+---
+
+The `geo.shapeData()` function restructures data with existing latitude and longitude
+fields to meet the requirements of the Geo package by renaming the existing latitude
+and longitude fields to **lat** and **lon** and adding an **s2_cell_id** tag.
+
+**`geo.shapeData()` does the following:**
+
+- renames existing latitude and longitude fields to `lat` and `lon`
+- pivots data into row-wise sets based on the [`correlatinKey`](#correlationkey)
+- uses the `lat` and `lon` values to generate the `s2_cell_id` tag based on a
+  specified [S2 cell level](https://s2geometry.io/resources/s2cell_statistics.html)
+- adds the `s2_cell_id` column to the group key
```

Comment:
maybe combine to simplify:
Use the `geo....` function to do the following:
- Rename existing...
- Pivot...
- Generate `s2..` using `lat` and `lon` values